packages/formation-react/README.md

```diff
@@ -25,15 +25,15 @@ From the project root, run the following commands:
 - AdditionalInfo.js
 - AlertBox.js
 - Breadcrumbs.js
-- ErrorableCheckbox.js
-- ErrorableDate.js
-- ErrorableFileInput.js
-- ErrorableMonthYear.js
-- ErrorableNumberInput.js
-- ErrorableRadioButtons.js
-- ErrorableSelect.js
-- ErrorableTextArea.js
-- ErrorableTextInput.js
+- Checkbox.js
+- Date.js
+- FileInput.js
+- MonthYear.js
+- NumberInput.js
+- RadioButtons.js
+- Select.js
+- TextArea.js
+- TextInput.js
```

Comment:
This gets reflected in the `Visual Design` sidebar in the gatsby site, right?

Reply:
Ooh, good question. I doubt it. :grimacing: